pkg/sql/schemachanger/scplan/scplan_test: TestExplainPlanIsMemoryMonitored failed

pkg/sql/schemachanger/scplan/scplan_test.TestExplainPlanIsMemoryMonitored [failed](https://tanzanite.cluster.engflow.com/invocations/default/ac423191-0c82-4544-a53f-a630186a97ff?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc3FsL3NjaGVtYWNoYW5nZXIvc2NwbGFuOnNjcGxhbl90ZXN0) on release-24.3 @ [58b09793ce98dfaaefbe3577d19703931a781181](https://github.com/cockroachdb/cockroach/commits/58b09793ce98dfaaefbe3577d19703931a781181):

Fatal error:

```
panic: test timed out after 53m20s
running tests:
	TestExplainPlanIsMemoryMonitored (46m0s)

```
Stack: 

```
goroutine 275132 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x265
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x45
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestExplainPlanIsMemoryMonitored
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestExplainPlanIsMemoryMonitored2433381333
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExplainPlanIsMemoryMonitored.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-44270

--- a/pkg/sql/schemachanger/scplan/BUILD.bazel
+++ b/pkg/sql/schemachanger/scplan/BUILD.bazel
@@ -69,6 +69,7 @@ go_test(
         "//pkg/sql/sem/tree",
         "//pkg/testutils/datapathutils",
         "//pkg/testutils/serverutils",
+        "//pkg/testutils/skip",
         "//pkg/testutils/sqlutils",
         "//pkg/util/leaktest",
         "//pkg/util/log",

--- a/pkg/sql/schemachanger/scplan/plan_test.go
+++ b/pkg/sql/schemachanger/scplan/plan_test.go
@@ -30,6 +30,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
 	"github.com/cockroachdb/cockroach/pkg/testutils/datapathutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
+	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
@@ -247,6 +248,8 @@ func marshalOps(t *testing.T, ts scpb.TargetState, stages []scstage.Stage) strin
 func TestExplainPlanIsMemoryMonitored(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
+
+	skip.UnderDuress(t, "large test; uses a lot of memory")
 
 	ctx := context.Background()
 	s, db, _ := serverutils.StartServer(t, base.TestServerArgs{

